### ISSUE TITLE
update of portable version 4.0.2 and 4.0.3

### ISSUE THREAD

**Issue by faon-92** [SubtitleEdit/plugins#240]
Hi everyone,
I'm used to work with portable version, but since 4.0.1 when I update by copying Settings.xml to the new version folder, it does not work any more: when I double click the executable file, the screen goes wrong (see screen capture).
Any idea?

many thanks
<img width="1280" alt="SubtitleEdit_0ldtCgcOja" src="https://github.com/SubtitleEdit/plugins/assets/127494540/37517735-c248-4c10-b4e8-cd72832e1b2d">

**Comment by niksedk** [comment]
Try toggling "Dark theme" on/off and see what happens... Options - Setting - Appearance

**Comment by faon-92** [comment]
on SE4.0.3 portable, when I open Options-settings, I can not read the user interface (see screen capture)
<img width="1280" alt="SubtitleEdit_JhhJLbzi54" src="https://github.com/SubtitleEdit/plugins/assets/127494540/f23e5f45-a933-45b9-ae6c-1fde2f210817">

**Comment by faon-92** [comment]
Yes, you are right, I managed to tick off the dark theme on 4.0.3 and it works normally then.
thanks a lot for the help
